app/models/service_provider_certificate.rb

```diff
@@ -18,4 +34,12 @@ def self.expiration_time_to_colorized_s(time)
   def self.warning_period
     (Figaro.env.certificate_expiration_warning_period || 60).to_i.days.from_now
   end
+
+  def expiration_css_class
+    if not_after < Time.zone.now
+      'certificate-expired'
+    elsif not_after < self.class.warning_period
+      'certificate-warning'
+    end
```

Comment:
If we want to retain the previous colors while avoiding the inline style, maybe we could use [design system color utility classes](https://designsystem.digital.gov/utilities/color/) and avoid the custom classes? This would also give us some color consistency for free. 

```suggestion
    if not_after < Time.zone.now
      'bg-red text-white'
    elsif not_after < self.class.warning_period
      'bg-yellow text-ink'
    end
```

Reply:
Went with the more specific styles approach re: https://github.com/18F/identity-dashboard/pull/404/files#r604387482